docker-compose.yml

```diff
@@ -7,16 +7,32 @@ services:
       - "5432:5432"
     environment:
       POSTGRESQL_ADMIN_PASSWORD: mysecretpassword
-    networks:
-      - default
   core:
     image: fabric8-services/fabric8-wit:latest
-    command: -config /usr/local/wit/etc/config.yaml
+#    command: -config /usr/local/wit/etc/config.yaml
     environment:
-      F8_POSTGRES_HOST: db
+      F8_AUTH_URL: "http://localhost:8089"
+      F8_DEVELOPER_MODE_ENABLED: "true"
     ports:
       - "8080:8080"
-    networks:
-      - default
+    network_mode: "host"
     depends_on:
+      - auth
+  db-auth:
+    image: registry.centos.org/postgresql/postgresql:9.6
+    ports:
+      - "5433:5432"
+    environment:
+      POSTGRESQL_ADMIN_PASSWORD: mysecretpassword
+  auth:
+    image: docker.io/fabric8/fabric8-auth:latest
+#    command: -config /usr/local/auth/etc/config.yaml
+    environment:
+      AUTH_WIT_URL: "http://localhost:8080"
+      AUTH_DEVELOPER_MODE_ENABLED: "true"
+    ports:
+      - "8089:8089"
+    network_mode: "host"
+    depends_on:
+      - db-auth
       - db
```

Comment:
does `auth` depend on both `db` and `db-auth` ?

Reply:
technically auth doesn't depend on db. But by adding such dependency we can up all dependencies by docker-compose up -d auth

Reply:
I changed it a bit - https://github.com/fabric8-services/fabric8-wit/pull/1668/commits/c1dafe88861cb628dbbf7283212c646d20a16a39